tests/interfaces/records/handlers/records-query.spec.ts

```diff
@@ -742,6 +741,82 @@ describe('RecordsQueryHandler.handle()', () => {
         const plaintextBytes3 = await DataStream.toBytes(plaintextDataStream3);
         expect(Comparer.byteArraysEqual(plaintextBytes3, bobMessageBytes)).to.be.true;
       });
+
+      it('should only be able to decrypt record with a correct derived private key - `schemas` derivation scheme', async () => {
+        // scenario: Alice writes into her own DWN an encrypted record and she is able to decrypt it
+
+        const alice = await TestDataGenerator.generatePersona();
+        TestStubGenerator.stubDidResolver(didResolver, [alice]);
+
+        // encrypt Alice's record
+        const originalData = TestDataGenerator.randomBytes(1000);
+        const originalDataStream = DataStream.fromBytes(originalData);
+        const dataEncryptionInitializationVector = TestDataGenerator.randomBytes(16);
+        const dataEncryptionKey = TestDataGenerator.randomBytes(32);
+        const encryptedDataStream = await Encryption.aes256CtrEncrypt(dataEncryptionKey, dataEncryptionInitializationVector, originalDataStream);
+        const encryptedDataBytes = await DataStream.toBytes(encryptedDataStream);
+
+        const encryptionInput: EncryptionInput = {
+          initializationVector : dataEncryptionInitializationVector,
+          key                  : dataEncryptionKey,
+          keyEncryptionInputs  : [{
+            derivationScheme : KeyDerivationScheme.Schemas,
+            publicKey        : alice.keyPair.publicJwk // reusing signing key for encryption purely as a convenience
+          }]
+        };
+
+        const { message, dataStream } = await TestDataGenerator.generateRecordsWrite({
+          requester : alice,
+          data      : encryptedDataBytes,
+          encryptionInput
+        });
+
+        const bobWriteReply = await dwn.processMessage(alice.did, message, dataStream);
+        expect(bobWriteReply.status.code).to.equal(202);
```

Comment:
```suggestion
        const writeReply = await dwn.processMessage(alice.did, message, dataStream);
        expect(writeReply.status.code).to.equal(202);
```
since we don't have a bob in this test scenario

Reply:
Sloppy me!!